packages/cxl-ui/package.json

```diff
@@ -10,24 +10,25 @@
     "directory": "packages/cxl-ui"
   },
   "dependencies": {
-    "@conversionxl/cxl-lumo-styles": "^1.4.4"
+    "@conversionxl/cxl-lumo-styles": "^1.4.4",
+    "@vaadin/details": "^23.3.7"
```

Comment:
Should also be devDependency.

For time being, I run with this uncommitted patch

```diff
diff --git a/.eslintrc b/.eslintrc
index 38dbff3f..aa4c657f 100644
--- a/.eslintrc
+++ b/.eslintrc
@@ -16,6 +16,7 @@
       }
     ],
     "import/extensions": "off",
+    "import/no-extraneous-dependencies": "off",
     "import/prefer-default-export": "off",
     "no-restricted-globals": "off",
     "no-underscore-dangle": "off"
```